testing/h1-cli/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Adam Dobrawy <a.dobrawy@hyperone.com>
+# Maintainer: Adam Dobrawy <a.dobrawy@hyperone.com>
+pkgname=h1-cli
+_pkgname=${pkgname/-cli/}
+pkgver=1.5.1
+pkgrel=0
+pkgdesc="Command-line interface for HyperOne platform"
+url="https://www.hyperone.com/tools/cli/"
+arch="x86_64"
+license="MIT"
+depends="openssh-client"
+makedepends="nodejs npm"
+subpackages="rbx-cli:rbx"
+source="$pkgname-$pkgver.tar.gz::https://github.com/hyperonecom/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!strip"
+
+build() {
+   cd "$builddir"
```

Comment:
`cd "$builddir"` is redundant nowadays, this could be removed

Reply:
Fixed.